client/my-sites/checkout/composite-checkout/composite-checkout.js

```diff
@@ -301,14 +301,12 @@ export default function CompositeCheckout( {
 	) => {
 		const getIsFieldDisabled = () => isDisabled;
 		return (
-			<ContactDetailsFormFields
-				countriesList={ countriesList }
```

Comment:
This prop is marked as required in the PropTypes -- do we not need to pass it here?

Reply:
nvm - I just saw the default export, which pulls this from redux